manim/animation/transform.py

```diff
@@ -200,7 +201,7 @@ def __init__(self, mobject, methods):
 class ApplyMethod(Transform):
     def __init__(
         self, method: types.MethodType, *args, **kwargs
-    ) -> None:  # method typing? for args?
+    ) -> None:  # method typing (we want to specify Mobject method)? for args?
```

Comment:
I have more information about this. types.methodType is an abstract method, and not a method bound to an object. We need the type for a bound method which is a Callable[...] and has a __self__ mobject.
However it looks like it might not be possible to do this correctly
https://stackoverflow.com/questions/64225522/python-typing-how-to-type-hint-a-variable-as-a-bound-method
https://stackoverflow.com/questions/58085648/mypy-type-annotation-for-a-bound-method